.github/settings.yml

```
@@ -41,7 +41,8 @@ collaborators:
 
   # English approvers (approver for all files in repository)
 
-  
+  - username: nate-double-u
+    permission: admin
```

```suggestion
    permission: push
```
I guess `push` is enough permission unless you are interested in maintaining (change setting, managing branches, .. ) the repository itself. :)

Even though you already have a technical access to this repository already.

According to your decision, it'd better update https://github.com/cncf/people/blob/main/config.yaml#L493
as well. (admin->write) :)

---

I'm happy to help with maintenance of the repo itself (I already do that a lot with other projects, so am happy to expand my umbrella to cover this too if needed). I set the permissions in this repos settings file to be in line with the reality of being an admin at the CNCF level. I don't currently get all the notifications, so I'm hoping the bot will see the codeowners listing and ping based on that.

All that said, I'm happy to reduce the permissions in both places—I don't want to overstep.

---

I'm also happy to add myself to the list of owners and maintainers (option one above) if that helps keep things consistent.

---

Thanks! It would be nice if you can help! :)